app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -18,7 +18,25 @@ def textual_group_relationships
   end
 
   def textual_group_status
-    %i(refresh_status)
+    textual_authentications + %i(refresh_status)
+  end
+
+  def textual_authentications
+    authentications = @ems.authentications
+    return [{:label => _("Default Authentication"), :title => t = _("None"), :value => t}] if authentications.blank?
+
+    authentications.collect do |auth|
+      label =
+        case auth.authtype
+        when "default" then _("Default")
+        when "bearer" then _("Bearer")
```

Comment:
isn't here any option to just capitalize the authtype? this is was is practically done anyway, but manually.